domains/hettpatell.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "patelhettt"
+    },
+    "records": {
+        "A": "216.24.57.1"
```

"A" records is an Array of string
```suggestion
        "A": ["216.24.57.1"]
```